lua/effects/powerups_groundpound_shockwave_huge/init.lua

```diff
@@ -0,0 +1,91 @@
+-- Slimmed down version of m9k_gdcw_cinematicboom from M9K
+
+function EFFECT:Init( data )
+    self.Pos = data:GetOrigin() -- Origin determines the global position of the effect
+    self.Scale = data:GetScale() -- Scale determines how large the effect is
+    self.DirVec = data:GetNormal() -- Normal determines the direction of impact for the effect
+    self.Emitter = ParticleEmitter( self.Pos ) -- Emitter must be there so you don't get an error
+
+    sound.Play( "ambient/explosions/explode_" .. math.random( 1, 4 ) .. ".wav", self.Pos, 100, 100 )
+
+    self:Dust()
+
+    self.Emitter:Finish()
+end
+
+function EFFECT:Dust()
```

Comment:
from experience, this many effects and emitters can kill weaker pcs

Reply:
It's the same as `m9k_gdcw_cinematicboom` (which gets used by a variety of things in m9k) without the 'Debrizzle' particles, which increase in quantity with `Scale * Magnitude`.

However, m9k usually leaves Scale (which affects the other particle counts) in the range of 1-2, only using 4 for the ammo crate explosions. I'll take a stab ad reducing the particle amount.

Reply:
Alright. If you can't reasonably test its performance on other setups, we should just keep a close eye out for anyone complaining about performance while it's being used 👍